src/main/java/Duke.java

```diff
@@ -1,10 +1,105 @@
+import java.util.Iterator;
+import java.util.Scanner;
+import java.util.Vector;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        SystemMessages.welcomeMessage();
+
+        boolean breakCheck = false;
+        String input,firstWord;
+        Vector<Task> dataStorage = new Vector<>();
+        int counter = 1;
+        Scanner in = new Scanner(System.in);
+
+        while(!breakCheck) {
+            input = in.nextLine().trim();
+            firstWord = input.split(" ",2)[0];
```

Comment:
These indexes might be a little hard to understand as magic numbers so you may want to make a constant to describe them better

Reply:
Same thing for some of messages that are printed, might be better to make them constants?